### ISSUE TITLE
feat: Act also as a snapcast client

### ISSUE THREAD

**Issue by konikvranik** [music-assistant/companion#55]
I use Snapcast in my home, and it would be nice if Companion could act as a Snapcast client too, in addition to the Squeezelite.

**Comment by arctixdev** [comment]
Yes I agree that this would be very useful. I don't have time to work on it at the moment but anyone is welcome to pick up the work 🙂
